tools/wpt/expectation.json

```diff
@@ -13034,8 +13032,7 @@
     },
     "wrapKey_unwrapKey": {
       "wrapKey_unwrapKey.https.any.html": false
-    },
-    "randomUUID.any.html": true
```

Comment:
Where did this test go?

Reply:
Oh it is `randomUUID.https.any.html` now.

Reply:
It was renamed to `randomUUID.https.any.js` in https://github.com/web-platform-tests/wpt/pull/28967. Looks like we added the renamed version but didn't remove the old name.